PocketKit/Sources/PocketKit/Home/SectionHeaderView.swift

```diff
@@ -80,6 +86,17 @@ extension SectionHeaderView {
         updateButtonConfiguration(with: model.buttonTitle, and: model.buttonAction)
     }
     
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        myListButton.configurationUpdateHandler = { button in
+            var config = button.configuration
+            config?.image = UIImage(asset: .chevronRight)
+                .withTintColor(UIColor(.ui.lapis1), renderingMode: .alwaysOriginal)
+                .resized(to: SectionHeaderView.buttonImageSize)
+            button.configuration = config
```

Comment:
Wondering why this code needs to live within `traitCollectionDidChange` and not in `updateButtonConfiguration`?

Reply:
updateButtonConfiguration wasn't being called when switching from dark mode to light mode (and vice versa) so the button colors weren't being updated

Reply:
discussed with @jacobsimeon, the issue was that we were resizing after applying tint color